src/components/about-us/utils/pop-up-panel.js

```diff
@@ -0,0 +1,116 @@
+import { screen } from './screen'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+const defaultZIndex = 2
+
+const Panel = styled.div`
+  z-index: calc(${defaultZIndex} + 1);
+  position: fixed;
+  overflow: scroll;
+  background: ${props => props.background};
+  -webkit-overflow-scrolling: touch;
+  ${screen.mobile`
+    width: ${props => props.mobWidth};
+    height: ${props => props.mobHeight};
+    top: ${props => props.mobPositionTop}; 
+    bottom: ${props => props.mobPositionBottom}; 
+    left: ${props => props.mobPositionLeft};
+    right: ${props => props.mobPositionRight};
+  `}
+  ${screen.tablet`
+    width: ${props => props.tabletWidth};
+    height: ${props => props.tabletHeight};
+    top: ${props => props.tabletPositionTop}; 
+    bottom: ${props => props.tabletPositionBottom}; 
+    left: ${props => props.tabletPositionLeft};
+    right: ${props => props.tabletPositionRight};
+  `}
+`
+
+class PopUpBox extends React.PureComponent {
+  constructor(props) {
+    super(props)
+    this.startY = 0
+    this.panel = null
+    this.handlePreventTouchstartWhenPanning = this._handlePreventTouchstartWhenPanning.bind(this)
+    this.handlePreventTouchendWhenPanning = this._handlePreventTouchendWhenPanning.bind(this)
+    this.handlePreventTouchmoveWhenPanning = this._handlePreventTouchmoveWhenPanning.bind(this)
+  }
+
+  componentDidMount() {
+    window.document.body.addEventListener('touchstart', this.handlePreventTouchstartWhenPanning, {
+      passive: false
+    })
+    window.document.body.addEventListener('touchend', this.handlePreventTouchendWhenPanning, {
+      passive: false
+    })
+    window.document.body.addEventListener('touchmove', this.handlePreventTouchmoveWhenPanning, {
+      passive: false
+    })
+  }
+
+  componentWillUnmount() {
+    this.panel = null
+    window.document.body.removeEventListener('touchstart', this.handlePreventTouchstartWhenPanning, {
+      passive: false
+    })
+    window.document.body.removeEventListener('touchend', this.handlePreventTouchendWhenPanning, {
+      passive: false
+    })
+    window.document.body.removeEventListener('touchmove', this.handlePreventTouchmoveWhenPanning, {
+      passive: false
+    })
+  }
+
+  _handlePreventTouchstartWhenPanning(event) {
+    this.startY = event.touches[0].screenY
+  }
+
+  _handlePreventTouchendWhenPanning(event) {
+    this.panel.scrollTop = this.panel.scrollTop + (this.startY - event.changedTouches[0].screenY)
+  }
+
+  _handlePreventTouchmoveWhenPanning = (event) => {
+    event.preventDefault()
+    this.panel.scrollTop = this.panel.scrollTop + (this.startY - event.changedTouches[0].screenY)
+    this.startY = event.touches[0].screenY
+  }
```

Comment:
We may need to figure out the difference between `touches` and `changedTouches`.

Reply:
![2018-07-23 6 35 00](https://user-images.githubusercontent.com/6375655/43072034-3d23d6e4-8ea7-11e8-8769-3ed978d4dad3.png)

In our case, `touches` and `changedTouches` are exactly the same when `touchmove` event is triggered.